### PR TITLE
made changes to the json output format

### PR DESCRIPTION
**Pull request by cornzyblack**
Made a pull request

### CODE REVIEW

**Comment by cornzyblack** [comment]
Alright, I did this for the images in this folder and not for OCR folder

**Comment by atherdon** [comment]
understood, will review it carefully tomorrow and then share to you my feedback. Nice work!

**Comment by atherdon** [comment]
I'm a bit confused(it was a while when i into this project) - can you specify to me(links, please) what images did you convert - so i also can compare a raw image with data, that we get in text.json @cornzyblack

**Comment by cornzyblack** [comment]
I did not modify the image processing method. I left it as it has originally been, the only thing I added was to convert its output to the desired JSON format.

The links for the images are, 

https://github.com/GroceriStar/ocr-grocery-parsing/blob/master/36867973-90dd0432-1d9f-11e8-9f14-51fd24ec842f.png
https://github.com/GroceriStar/ocr-grocery-parsing/blob/master/image2.png
https://github.com/GroceriStar/ocr-grocery-parsing/blob/master/image3.png

Thanks

**Comment by atherdon** [comment]
Ok, so let's open the first image(flat belly foods). @cornzyblack 
i see few list of ingredients on this image, but when i open test.json, it didn't match completely. please advice this. i mean it's not clear for me what data related to this image and we have also a "noise", that sometimes not an ingredient.

**Comment by cornzyblack** [comment]
So like I said, I did not modify the Image processing method and so this output is exactly as it has been before I touched it. When I first set up the codebase on my system and ran it, I got the same output you are getting. The only new thing I did was to split the former output into Key value pairs. 

While I look for ways on how to fix this issue, is the developer who wrote the code still available to talk to?

**Comment by atherdon** [comment]
ok, i got it. Can your solution be adjusted in order to return less 'noisy' data? Because right now - we're still in the middle: i cannot pass this data to our Javascript developers.

I can buzz our last developer, that was working on this script. But i'm not sure if he will have time. 
Can you tell me what exactly you want to ask? I mean can you specify what the problem is? is it about the installation process? Please advice. Maybe I can forward your questions to other team members - but I need to know details.


@sadegh1404 sorry for bothering you, we're stuck and need some help from you. will you be able to help us?

File: OCR_grocery.py [hunk 1]
```diff
@@ -27,20 +27,30 @@ def spellchecker(misspelled_word='caaar',n_gram = 1):
     return min_word
 
 
+def ingr_splitter(item):
+    search_obj = re.search('\d+', item)
+    if (search_obj != None):
+        quantity = item[search_obj.start() : search_obj.end()]
+        ingredient = item[search_obj.end():]
+    else:
+        quant = 1
```
Comment (atherdon) [inline OCR_grocery.py]: at `if` statement we have `quantity` but at `else` we have `quant` is this is a mistake? please advice

File: OCR_grocery.py [hunk 2]
```diff
@@ -101,7 +112,8 @@ def spellchecker(misspelled_word='caaar',n_gram = 1):
 		#         q_measure.append(s)
 
 	print('write to json')
-	json_data.append({'words':checked_words,'quntity&measurements':q_measure})
+	print(q_measure)
+	json_data.append(q_measure)
```
Comment (atherdon) [inline OCR_grocery.py]: can you keep an old version?
just add comments, aka
``` # json_data.append({'words':checked_words,'quntity&measurements':q_measure}) ```

**Comment by sadegh1404** [comment]
hello @atherdon  , yes of course .what's the problem ?

**Comment by atherdon** [comment]
@cornzyblack this is @sadegh1404 - he was a contributor that worked on this project before you

btw, in order to make this whole thing more clear, i created and published a set of articles.
i understand that both of you aware about this, but i wanted to put everything in one place
1. https://medium.com/groceristar/ocr-project-part1-simple-introduction-ab7b64bf1864
2. https://medium.com/groceristar/ocr-project-part2-details-85739239f42
3. https://medium.com/groceristar/ocr-project-part3-links-collection-2cfc58904caf

**Comment by atherdon** [comment]
i think this PR shoudl be closed, cause we change our direction drammatically

**Comment by atherdon** [comment]
@cornzyblack can ou revert your changes that you've made at Jan 1st?
maybe it'll be better if you re-fork main repo again at separated folder and then push there your code with food composition list? because right now i cannot merge it. First part(from Jan 1st) will conflict with our main repo. Please advice

**Comment by atherdon** [comment]
if you'll need help with it - tell me.
i think its also a good task - improving skill to working together at git project

**Pull request closed without merging**